site/docs/reference/Connectors/materialization-connectors/Rockset.md

```diff
@@ -42,4 +45,84 @@ materializations:
       	workspace: ${namespace_name}
       	collection: ${table_name}
     source: ${tenant}/${source_collection}
-```
\ No newline at end of file
+```
+## Bulk ingestion for large backfills of historical data
+
+You can backfill large amounts of historical data into Rockset using a *bulk ingestion*. Bulk ingestion must originate in S3 and requires additional steps in your dataflow. Flow's Rockset connector supports this through the GitOps workflow.
+
+### Prerequisites
+
+Before completing this workflow, make sure you have:
+* A working catalog spec including at least one Flow collection with its schema specified.
+* A production or [development](../../../getting-started/installation.md) environment
+::: tip
+The following is an intermediate workflow. As needed, refer to [this guide](../../../guides/create-dataflow.md) for the basic steps to create and deploy a catalog spec using the GitOps workflow.
+:::
+
+### How to configure a bulk ingestion
```

Comment:
nit: I think "perform" may be a better word than "configure" here, since this is a process that requires human interaction at several points, rather than someone you set and forget.